app/data/projects.json

```diff
@@ -141,6 +141,13 @@
     "projectWebsite": "http://picketlink.org/",
     "category": "Development"
   },
+  {
+    "projectName": "Python",
+    "projectDescription": "The Python programming language",
+    "projectRepository": "https://github.com/picketlink",
```

Comment:
Copy and paste error, the repository is https://github.com/python

Reply:
Oops! Fixed.